DasharoPayloadPkg/BlSupportDxe/BlSupportDxe.c

```diff
@@ -226,6 +226,11 @@ BlDxeEntryPoint (
   EFI_HOB_GUID_TYPE                       *GuidHob;
   SYSTEM_TABLE_INFO                       *SystemTableInfo;
   EFI_PEI_GRAPHICS_INFO_HOB               *GfxInfo;
+  EFI_GUID                                FwGuid;

```

Comment:
@SergiiDmytruk please extract changes to `BlSupportDxe` to separate commit, we will drop it eventually so having dedicated commit would make it easier.

Reply:
a5428a73362b2d25a1c220a8582f3a80eae04013 709130a9b456ed4e3836a269a8fa7239fd73b1ec